.github/dependabot.yml

```diff
@@ -68,11 +68,29 @@ updates:
   - mtoonen
   - mprins
   labels:
-  - dependencies
   - javascript
   - build infrastructure
   - backport v5.8.x
 
+- package-ecosystem: npm
+  directory: "/tailormap-components"
+  schedule:
+    interval: weekly
+    day: wednesday
+    timezone: Europe/Amsterdam
+  open-pull-requests-limit: 25
+  reviewers:
+    - mtoonen
```

Comment:
Je mag mij erbij zetten, maar dat is nog maar beperkt nuttig ;-)
Waarom niet alle devs erbij? Iedereen gaat er toch aan ontwikkelen?

Reply:
ja of niemand, er kijkt (of reageert) nu ook geen hond naar zijn notificaties